registry/coder-labs/modules/copilot-cli/README.md

```diff
@@ -0,0 +1,213 @@
+        command     = "npx"
+        args        = ["-y", "@playwright/mcp@latest", "--headless", "--isolated"]
+        description = "Browser automation for testing and previewing changes"
+        name        = "Playwright"
+        timeout     = 5000
+        type        = "local"
+        tools       = ["*"]
+        trust       = false
+      }
+    }
+  })
+
+  pre_install_script = <<-EOT
+    #!/bin/bash
+    curl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash -
+    sudo apt-get install -y nodejs
+  EOT
+}
+```
+
+> [!NOTE]
+> GitHub Copilot CLI does not automatically install MCP servers. You have two options:
+>
+> - Use `npx -y` in the MCP config (shown above) to auto-install on each run
+> - Pre-install MCP servers in `pre_install_script` for faster startup (e.g., `npm install -g @modelcontextprotocol/server-filesystem`)
+
+### Direct Token Authentication
+
+```tf
+module "copilot_cli" {
+  source       = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version      = "0.1.0"
+  agent_id     = coder_agent.example.id
+  workdir      = "/home/coder/project"
+  github_token = "your_github_token_here" # Or use data.coder_external_auth.github.access_token
+}
+```
+
+### Standalone Mode
+
+Run and configure Copilot CLI as a standalone tool in your workspace.
+
+```tf
+module "copilot_cli" {
+  source       = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version      = "0.1.0"
+  agent_id     = coder_agent.example.id
+  workdir      = "/home/coder"
+  report_tasks = false
+  cli_app      = true
+}
+```
+
+### Custom Configuration
+
+You can customize the entire Copilot CLI configuration:
+
+```tf
+module "copilot_cli" {
+  source   = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version  = "0.1.0"
+  agent_id = coder_agent.example.id
+  workdir  = "/home/coder/projects"
+
+  copilot_config = jsonencode({
+    banner = "auto"
+    theme  = "dark"
+    trusted_folders = [
+      "/home/coder/workspace",
+      "/home/coder/projects"
+    ]
+  })
+}
+```
+
+## Authentication
+
+The module supports multiple authentication methods (in priority order):
+
+1. **Direct Token** - Pass `github_token` variable (OAuth or Personal Access Token)
+2. **Coder External Auth** - Automatic if GitHub external auth is configured in Coder
+3. **Interactive** - Copilot CLI prompts for login via `/login` command if no auth found
+
+> [!NOTE]
+> OAuth tokens work best with Copilot CLI. Personal Access Tokens may have limited functionality.
+
+## Session Resumption
+
+By default, the module resumes the latest Copilot CLI session when the workspace restarts. Set `resume_session = false` to always start fresh sessions.
+
+## Task Reporting
+
+When enabled (default), Copilot CLI can report task progress to the Coder UI using [AgentAPI](https://github.com/coder/agentapi). Custom MCP servers provided via `mcp_config` are merged with the Coder MCP server automatically.
+
+To disable task reporting, set `report_tasks = false`.
```

Comment:
I feel this is redundant as Task reporting is not a separate feature but part of Coder Tasks.